irssi/content.md

```diff
@@ -1,36 +1,44 @@
 # What is irssi?
-
-Irssi is a terminal based IRC client for UNIX systems. It also supports SILC
-and ICB protocols via plugins. Some people refer to it as 'the client of the
+# 
```

Comment:
Little things like this `#` are what I was referring to -- there's a couple more minor changes like this in this same file.